universal-login-wallet/src/ui/styles/components/_top-up.sass

```diff
@@ -197,21 +201,30 @@
 
     .copy-btn-feedback
       visibility: hidden
-      text-align: center
-      padding: 5px
-      z-index: 1
       opacity: 0
-      color: green
+      position: absolute
+      top: -15px
+      padding: 0px 5px
+      font-size: 12px
+      line-height: 18px
+      transition: visibility 0s 1s, opacity 1s linear
+      color: #FFFFFF
+      background-color: #2ec9f5
 
-    .copy-btn-feedback:before
-      content: '\2713'
+    .copy-btn-feedback::after
+      content: ''
```

Comment:
Seems like a lot of properties here are duplicated from `react/topUp.sass`. Do we actually need them here?

Reply:
Yes, it is desired because we want to have the same styles even when we remove the wallet from project and pass it to Jarvis

Reply:
Done